change/@ni-nimble-components-a8b6518e-7eb6-4450-8d81-4787ae2d1874.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should we document the behavior in storybook? Maybe in the table styling section?

https://60e89457a987cf003efc0a5b-gzrnmbkymk.chromatic.com/?path=/docs/components-table--docs#styling

Might refer to it as "full-bleed" table styling to align with the text-field frameless full-bleed mode

Reply:
I think it'd be good to do so. The main benefit for me is that clients might want this appearance but not realize they have to explicitly disable grouping (rather than just not configuring grouping).